joss/paper.md

```diff
@@ -46,21 +46,25 @@ According to the package documentation, it is possible to summarize some of the
 The following table shows some characteristics that differentiate each package.
 | Characteristic                                | Copulas.jl         | DatagenCopulaBased.jl         | BivariateCopulas.jl          |
 |-----------------------------------------------|--------------------|--------------------|--------------------|
-| Every Archimedean Copula sampling     | Yes                 | No                 | No                 |
-| Multivariate Copula sampling | Yes                | No                 | Yes                 |
-| Nested Copula Sampling             | No                 | No                 | Yes    |
-| Fitting Copula | Yes | only bivariate case | No |
+| Every Archimedean Copula sampling             | Yes                | No                 | No                 |
+| Obscure Bivariate Copulas                     | Yes                | No                 | No                 |
+| Classic Bivariate copulas sampling            | Yes                | Yes                | Yes                |
+| Archimedean copulas                           | All                | Classic only       | Classic only       |
+| Multivariate Copula                           | Yes                | Yes                | No                 |
+| Archimedean Chains                            | No                 | Yes                | No                 |
+| Fitting Copula                                | No                 | No                 | Yes                |
+| Plotting Copula                               | No                 | No                 | Yes                |
 ##  Efficiency
-To perform an efficiency test we use the "BenchmarkTools" package with the objective of comparing the execution time and the amount of memory necessary to generate copula samples with each package. We generate 10^6 samples for Clayton copula of dimensions 2, 5, 10 with parameter 0.8
+To perform an efficiency test we use the [`BenchmarkTools.jl`](https://github.com/JuliaCI/BenchmarkTools.jl) [@BenchmarkTools] package with the objective of comparing the execution time and the amount of memory necessary to generate copula samples with each package. We generate 10^6 samples for Clayton copula of dimensions 2, 5, 10 with parameter 0.8

```

Comment:
For the reference to work you need to add it to the .bib file, taking it from their CITATION file there : https://github.com/JuliaCI/BenchmarkTools.jl/blob/master/CITATION.bib

@ARTICLE{BenchmarkTools,
  author =	 {{Chen}, Jiahao and {Revels}, Jarrett},
  title =	 "{Robust benchmarking in noisy environments}",
  journal =	 {arXiv e-prints},
  keywords =	 {Computer Science - Performance, 68N30, B.8.1, D.2.5},
  year =	 2016,
  month =	 "Aug",
  eid =		 {arXiv:1608.04295},
  archivePrefix ={arXiv},
  eprint =	 {1608.04295},
  primaryClass = {cs.PF},
  adsurl =	 {https://ui.adsabs.harvard.edu/abs/2016arXiv160804295C},
  adsnote =	 {Provided by the SAO/NASA Astrophysics Data System}
}